ipmininet/install/build_openr-rc-20190419-11514.sh

```diff
@@ -0,0 +1,301 @@
+#!/bin/bash
+
+set -exo pipefail
+### Install packages for Debian-based OS ###
+
+apt-get update && apt-get install -yq autoconf-archive bison build-essential cmake curl flex git gperf joe libboost-all-dev libcap-dev libdouble-conversion-dev libevent-dev libgflags-dev libgoogle-glog-dev libkrb5-dev libpcre3-dev libpthread-stubs0-dev libnuma-dev libsasl2-dev libsnappy-dev libsqlite3-dev libssl-dev libtool netcat-openbsd pkg-config sudo unzip wget python3-venv python-setuptools python3-setuptools python-pip python3-pip ccache
```

Comment:
IIRC the install script does not guarantee that the host system is running debian/has apt-get and update-alternatives.
You should thus either disable OpenR (with a warning) on non-debian (likely on the python install script); or support the alternatives here.

Reply:
The OpenR install scripts in ipmininet currently support Debian/Ubuntu only:
https://github.com/cnp3/ipmininet/blob/master/ipmininet/install/__main__.py#L55

Reply:
I somehow missed that. Thanks!